acceptance-tests/src/test/java/io/blueocean/ath/offline/personalization/FavoritesCardsTest.java

```diff
@@ -100,6 +109,33 @@ public void testMultibranch() throws IOException, GitAPIException {
         dashboardPage.checkFavoriteCardStatus(fullNameMaster, SUCCESS);
         dashboardPage.checkFavoriteCardStatus(fullNameOther, SUCCESS);
 
+        //Check that favorite actions are hidden until pointer is hovering the row
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        Object backgroundColor;
+
+        backgroundColor = js.executeScript(
+            "return getComputedStyle(document.querySelectorAll('.actions-container')[0], ':after').getPropertyValue('background-color')"
```

Comment:
I feel like this could be made into something more reusable. Not needed for this PR but it would be nice

Reply:
Unfortunately I haven't found any other way of asserting this hover state css... if you have any idea I don't mind changing things

Reply:
How you are doing it is fine, but a java method that takes a selector and spits out computed css could be useful